src/components/Notification/index.tsx

```diff
@@ -0,0 +1,153 @@
+import * as React from 'react';
+import { Box, styled, useThemeProps } from '@mui/material';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faClose } from '@fortawesome/free-solid-svg-icons/faClose';
+import { faArrowUpRightFromSquare } from '@fortawesome/free-solid-svg-icons/faArrowUpRightFromSquare';
+import { shape } from '@balena/design-tokens';
+import { MUILinkWithTracking } from '../MUILinkWithTracking';
+import { IconButtonWithTracking } from '../IconButtonWithTracking';
+import { useEffect } from 'react';
+import { useAnalyticsContext } from '../../contexts/AnalyticsContext';
+
+export interface NotificationProps {
+	campaignId: string;
+	children: React.ReactNode;
+	illustration?: React.ReactNode;
+	title: string;
+	linkHref: string;
+	linkLabel: string;
+	onClose?: (event: React.SyntheticEvent<any>) => void;
+}
+const NotificationRoot = styled(Box, {
+	name: 'Notification',
+	slot: 'root',
+})<{ ownerState: NotificationProps }>(({ theme, ownerState }) => ({
+	alignItems: 'center',
+	backgroundColor: 'white',
+	borderRadius: shape.border_radius.md.value + 'px',
+	boxShadow: '0px 0px 8px 0px rgba(35, 68, 94, 0.3)', // TODO add a design token for box shadows
+	display: 'flex',
+	gap: theme.spacing(3),
+	maxWidth: 400,
+	padding: theme.spacing(3),
+	position: 'relative',
+	width: '100%',
+}));
+
+const NotificationIllustration = styled('figure', {
+	name: 'Notification',
+	slot: 'illustration',
+})(({ theme }) => ({
+	margin: 0,
+	width: 48,
+	minWidth: 48,
```

Comment:
Doesn't the first make the second pointless?

Reply:
I changed to `maxWidth` for clarity (and reduced the min width value)